dvc/command/destroy.py

```diff
@@ -25,7 +25,7 @@ def run(self):
                 )
 
             self.repo.destroy()
-        except Exception:  # noqa, pylint: disable=broad-except
+        except DvcException:
             logger.exception("failed to destroy DVC")
```

Comment:
Just to unify the behaviour among all of the commands. We'll also need to reconsider(probably remove) Captain Obvious messages like this one.